localisation/00_wc_vanilla_replacements.csv

```diff
@@ -292,4 +292,19 @@ EVTDESC_MNM_20081_CHILD;My fellows[Root.GetInSociety] have managed to insert the
 EVTDESC_MNM_20083;My competent brethren[Root.GetInSociety] managed to snatch [From.GetTitledFirstNameNoRegnal] from [From.GetPossPronoun] guards and bring [From.GetObjectPronoun] to me in a sack!;;;;;;;;;;;;;x
 EVTDESC_MNM_20084;My incompetent brethren[Root.GetInSociety] failed to capture [From.GetTitledFirstNameNoRegnal], though apparently they slew a guard and a servant in the attempt. Even worse, it appears that [From.GetFirstName] has started to suspect my involvement in the ambush.;;;;;;;;;;;;x;
 stop_abduct_decision_desc;Call off my [Root.GetSocietyMembers] from abducting my current target.;;;;;;;;;;;;;x
-bloodthirsty_mass_sacrifice_decision;Hold Blood Rituals;;;;;;;;;;;;;x
\ No newline at end of file
+bloodthirsty_mass_sacrifice_decision;Hold Blood Rituals;;;;;;;;;;;;;xEVTDESC7880;[Root.PlotTarget.GetTitledName] will soon travel in a convoy on an isolated road. Perhaps if we gathered some mercenaries and disguised them as [Root.PlotTarget.GetHighwaymenType]... crime is on the rise, after all.;;;;;;;;;;;;;x
+EVTDESC7880;[Root.PlotTarget.GetTitledName] will soon travel in a convoy on an isolated road. Perhaps if we gathered some mercenaries and disguised them as [Root.PlotTarget.GetHighwaymenType]... crime is on the rise, after all.;;;;;;;;;;;;;x
+EVTDESC7882;My co-conspirator, [From.GetTitledName], has asked for gold to hire some mercenaries. They will ambush [Root.PlotTarget.GetTitledName] disguised as [Root.PlotTarget.GetHighwaymenType].;;;;;;;;;;;;;x
+EVTDESC7885;You are travelling in a convoy in the countryside of [Root.Location.GetName] when it comes to a sudden halt. The road has been blocked by a fallen tree, and the convoy leader explains that you will have to make a slight detour on an isolated smaller path.;;;;;;;;;;;;;x
+EVTDESC7886;After less than half a league on the small path, you are ambushed and surrounded by [Root.GetHighwaymenType]! They send your guards on their way, explaining that their interest lies with you alone. You offer the belongings you carry with you, but they merely laugh at this - it is your blood they want.;;;;;;;;;;;;;x
+EVTOPTA7887;Y-you're not [Root.GetHighwaymenType]!;;;;;;;;;;;;;x
```

Comment:
> You offer the belongings you carry with you, but they merely laugh at this - it is your blood they want.

Wow, *witch hunters* want your blood? What a surprise.

Reply:
Rewrote `EVTOPTA7887` in b840de76a319f1b39da43829bcbceb858ae14cd3.